app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Router for all users </h1>
\ No newline at end of file
+<% @users.each do |user| %>
+  <div clas='main-container'>
+    <div class='user-container'>
+      <img src="<%= user.photo %>" class='user-image'>
+    </div>
+    <h2 class='users-info'>
+      <%= link_to user.name, user_path(user.id) %>
+    </h2>
+    <p class='post-num'>Number of posts: <%= user.posts_counter%></p>
```

Comment:
- [Optional] Hi @josuerushanika, a better practice is to use Rails partial to render our codebase DRY. Instead of using @object.each(@users.each), we could have created a user partial(_user.html.erb) and rendered it when necessary instead of repeating ourselves. This piece of code is also found on the user show page, post index page